frontend/src/views/container/setting/index.vue

```diff
@@ -377,12 +369,12 @@ const handleIptables = () => {
         confirmDialogRefIptable.value!.acceptParams(params);
         return;
     } else {
-        iptablesVisible.value = true;
+        open.value = true;
     }
 };
 const onSubmitCloseIPtable = () => {
     save('IPtables', 'disable');
-    iptablesVisible.value = false;
+    open.value = false;
 };
 const onSubmitOpenIPtable = () => {
     save('IPtables', 'enable');
```

Comment:
The code is already quite clean and does not contain any errors, irregularities, or optimizations to suggest. However, I noticed that there were some small typos in the comments (e.g., "iptablesHelper2") which may be unintentional mistakes. Otherwise, no changes are needed based on these checks. Please ensure that the comments accurately reflect the functionality of each component.

If you're looking at it from an architectural standpoint or have more specific questions related to performance, testing, security considerations, etc., please let me know!